### ISSUE TITLE
JSON Schema no longer contains descriptions

### ISSUE THREAD

**Issue by fontoura** [openEHR/specifications-ITS-JSON#12]
Since PR #10 was merged, the JSON schemas at the OpenEHR website and here no longer contain descriptions for JSON types and attributes. Those were really useful for understaing the documentation.

**Comment by sebastian-iancu** [comment]
That's true and it is intentional. In one of the last SEC meeting it was decided to go further with current JSON schema (generated with Archie from BMM) and obsolete the previous ones (generated from UML). The reason was that the current can really validate more types, due to solving polymorphic type issues (see this topic https://discourse.openehr.org/t/json-schema-and-openapi-current-state-and-how-to-progress/1385).
The openEHR BMM files does not contain all information and descriptions present in openEHR UML files, therefore the JSON schema don't have that yet those either. We were not aware of any use or demand for such info in JSON schema, so we (SEC) decided to go further with the current variant, for the benefits mentioned above. 
If you have requirements towards adding those descriptions back to JSON schema, please provide some details about the use-case (if possible) and start a thread in https://discourse.openehr.org/c/specifications/its/41 so that we can have a discussion - I'm pretty sure there will be people interested (at least @pieterbos).

**Comment by pieterbos** [comment]
The BMM files can actually contain documentation, but currently do not. If we get BMM files with the descriptions, the generator will automatically add all of them to the JSON Schema ones, and we can update them easily. However, that is quite a lot of work unless someone makes a UML to BMM generator.

Have you found the search field on https://specifications.openehr.org/ , where you can type a class name and go directly to the specification of that class? Are you still missing something there?
